src/js/view/popup/scheduleCreationPopup.js

```diff
@@ -244,20 +244,20 @@ ScheduleCreationPopup.prototype._toggleIsPrivate = function(target) {
 ScheduleCreationPopup.prototype._onClickSaveSchedule = function(target) {
     var className = config.classname('popup-save');
     var cssPrefix = config.cssPrefix;
-    var title;
-    var startDate;
-    var endDate;
-    var rangeDate;
-    var form;
-    var isAllDay;
+    var title, startDate, endDate, rangeDate, form, isAllDay;
 
     if (!domutil.hasClass(target, className) && !domutil.closest(target, '.' + className)) {
         return false;
     }
 
     title = domutil.get(cssPrefix + 'schedule-title');
-    startDate = new TZDate(this.rangePicker.getStartDate()).toLocalTime();
-    endDate = new TZDate(this.rangePicker.getEndDate()).toLocalTime();
+    startDate = new TZDate(this.rangePicker.getStartDate());
```

Comment:
I think `toLocalTime() and getTime()` should be changed in case of not using customTimezoneOffset. If the user doesn't use `customTimezoneOffset`, the two methods returns identical result.